### PR TITLE
Fix #6

### LINKED ISSUE #6: pi monitoring script enhancement needed.
Enhancements are needed for pi monitoring package:

1. Vehicle specific string shouldn't be in the topic name. Instead, we need to use namespace in the launch file so the vehicle name will be appended in front of the topic names

2. Since we also using this for Jetson, we should rename this package to computer_monitoring package or something more intuitive.

3. Self.check_device already give the device name. Then the code should construct the topic names based on the output of the device name. Therefore, we don't need a switch or if statement there.

### PR DESCRIPTION
**Pull request by tjsos**

### CODE REVIEW

File: computer_monitoring/src/log_ram_temp.py [hunk 1]
```diff
@@ -84,6 +82,6 @@ def collect_and_publish(self):
             self.rate.sleep()
     
 if __name__ == "__main__":
-    rospy.init_node("pi_monitor_node")
+    rospy.init_node("pi_monitor_node", anonymous=True)
```
Comment (mzhouURI) [inline computer_monitoring/src/log_ram_temp.py]: can you change the node name? computer_monitor_node or something

**Review by mzhouURI — CHANGES_REQUESTED** [review]
see the comments in the main function

File: computer_monitoring/src/log_ram_temp.py (hunk 1 above)
Comment (tjsos) [reply computer_monitoring/src/log_ram_temp.py]: The launch file would overwrite the name inside the script. That's why I didn't change it inside.
Anyway, new commit fixes it.

Comment (mzhouURI) [reply computer_monitoring/src/log_ram_temp.py]: I see. but it worth changing the name inside the node to a more general name similar to the package name.
The code looks good. 
I will merge.

**Pull request merged**